### ISSUE TITLE
Add support for footnotes

### ISSUE THREAD

**Issue by owenh000** [owenh000/asciidoctor-multipage#3]

**Comment by veselov** [comment]
I'm actually quite fond of footnotes, and at this point, the only problem I see with their support is that footnotes from internal sections show up on all the corresponding parent pages. Is that the core of the problem - filtering out footnotes that don't belong, and re-indexing them so there are no holes in their numbering on individually separated pages?

**Comment by owenh000** [comment]
That's correct. We should also check (and include in tests) a footnote inside a block style table cell (nested document) to make sure that works properly.

**Comment by ilmarh** [comment]
The problem is still there. Used 0.0.18 to generate multipage version of asciidoc book (400paged pdf).
Got nice look for online help, but footnotes messed up the whole thing.
I'm using nested includes, may they be the cause?

**Comment by owenh000** [comment]
>I'm using nested includes, may they be the cause?

This is probably not related to nested includes.

**Comment by owenh000** [comment]
This issue requires #46 to be done first.
